packages/cli/lib/sandboxes.js

```diff
@@ -35,6 +35,30 @@ function getAccountName(config) {
   return `${config.name} ${isSandbox ? sandboxName : ''}(${config.portalId})`;
 }
 
+function getHasDevelopmentSandboxes(parentAccountConfig) {
+  const config = getConfig();
+  const parentPortalId = parentAccountConfig.portalId;
+  for (const portal of config.portals) {
+    if (
+      (portal.parentAccountId !== null ||
+        portal.parentAccountId !== undefined) &&
+      portal.parentAccountId === parentPortalId &&
+      portal.sandboxAccountType &&
+      portal.sandboxAccountType === 'DEVELOPER'
+    ) {
+      return true;
+    }
+  }
+  return false;
+}
+
+function getDevSandboxLimit(message) {
```

Comment:
Again, I'm ok with this solution for now, but is there any way to return the sandbox limit from the BE? This function seems pretty brittle, especially if the BE decides to change the error message.

Reply:
Yep let me check with BE to see if we can add an extra prop or something to pull from

Reply:
Talked with BE and they've added a new context property we can pull from to lessen the dependency on a string!